simularium/simularium-viewer
# Feature/freeze camera

Problem
=======
The education module will not need camera controls because it's a 2d simulation 

Solution
========
I added a prop called `stationaryViewport` that defaults to `false`. 

NOTES/QUESTIONS:

1. I made the assumption here that this setting is unlikely to change for a simulation. It's set when the canvas is mounted and nothing would happen if the prop changed but the canvas isn't remounted. I can add the ability to have it react to the prop change, but it's not needed for the education simulation and so I'm partial to waiting until it's a requirement for something else but open to changing if other people feel differently. 
2. any ideas for a clearer prop name are welcome. I could also see the argument for having a `movableViewport` prop that defaults to `true`

## Type of change
Please delete options that are not relevant.

* New feature (non-breaking change which adds functionality)

Steps to Verify:
----------------
1. `npm start`
2. change `stationaryViewport` to true in the example viewer 
3. camera controls shouldn't work and nothing should happen when you click on an agent.
